docs/whats_new.rst

```diff
@@ -17,6 +17,250 @@ in specific versions.
+- Warn the user that bools are not supported for ``default_member_permissions``. (:issue:`520`)
+- Update the Guild Iterator to not get stuck in an infinite loop. (:issue:`526`)
+    - Add a missing import for the scheduled event user iterator.
+- Change the default guild :class:`.GuildSticker` limit to 5. (:issue:`531`)
+- Handle optional :class:`Locale` instances (no longer create an enum value). (:issue:`533`)
+- Update the type field handling for audit logs. (:issue:`535`)
+    - :attr:`AuditLogDiff.type` objects are no longer always :class:`ChannelType` instances.
+- Dispatch :func:`disnake.on_reaction_remove` for :class:`.Thread` instances. (:issue:`536`)
+- Update :attr:`Guild.bitrate_limit` to use the correct value for the ``VIP_REGIONS`` feature flag. (:issue:`538`)
+- Handle :class:`ThreadAutoArchiveDuration` instances for ``default_auto_archive_duration`` when editing channels. (:issue:`568`)
+- Assume that ``None`` is an empty channel name and keep ``channel.name`` a string. (:issue:`569`)
+- Remove the ``$`` prefix from ``IDENTIFY`` payload properties. (:issue:`572`)
+- Replace old application command objects in cogs with the new/copied objects. (:issue:`575`)
+- Fix opus function calls on arm64 macOS. (:issue:`620`)
+- Improve channel/guild fallback in resolved interaction data, using :class:`PartialMessageable` for unhandled/unknown channels instead of using ``None``. (:issue:`646`)
+- Check the type of the provided parameter when validating names to improve end-user errors when passing an incorrect object to slash command and option names. (:issue:`653`)
+- Make the :func:`.ext.commands.default_member_permissions` decorator always work in cogs. (:issue:`678`)
+- Fix :attr:`Spotify.start`, :attr:`Spotify.end`, :attr:`Spotify.duration` raising :exc:`KeyError` instead of returning ``None``, improve activity typing. (:issue:`685`)
+- Fixes message initialization failing with threads and no intents by explicitly checking we have a guild object where one is required. (:issue:`699`, :issue:`712`)
+- Fixed an issue where it would be possible to remove other features when enabling or disabling the ``COMMUNITY`` feature for a :class:`.Guild`. (:issue:`705`)
+- Fix invalid widget fields. (:issue:`736`)
+    - :attr:`Widget.invite_url` and :attr:`Widget.fetch_invite` are now optional.
+    - :attr:`WidgetMember.avatar` and :attr:`WidgetMember.activity` now work properly and no longer always raise an exception or return ``None``.
+- No longer use deprecated `@!` syntax for mentioning users. (:issue:`743`)
+- Fix creation of forum threads without :class:`Permissions.manage_threads`. (:issue:`746`)
+- Don't count initial message in forum threads towards :attr:`Thread.message_count` and :attr:`Thread.total_message_sent`. (:issue:`747`)
+- Fix edge case with parsing command annotations that contain a union of non-type objects, like ``Optional[Literal[1, 2, 3]]``. (:issue:`770`)
```

Comment:
```suggestion
- |commands| Fix edge case with parsing command annotations that contain a union of non-type objects, like ``Optional[Literal[1, 2, 3]]``. (:issue:`770`)
```
small nit - noticed this earlier, might as well fix it now

Reply:
https://github.com/DisnakeDev/disnake/pull/775/commits/27c168d36dfebebe4921405bfe456a8f6e455c1c